docs/JSON-RPC-API/Authentication.md

```diff
@@ -0,0 +1,88 @@
+# Authentication and Authorization 
+
+Authentication identifies a user based on a username and password. Authorization verifies whether the user has
+access to the JSON-RPC method they are requesting.  
+
+Pantheon uses [JWT tokens](https://jwt.io/introduction/) to authenticate JSON-RPC API requests. 
+
+!!! important 
+    Ensure all API requests are made over HTTPS.  
```

Comment:
It would be interesting to add a brief description of why we have this warning here. Something on the lines:

> It is important that authenticated request to the nodes are made over an encrypted channel (HTTPS).  This prevents someone from eavesdropping the connection and stealing the JWT token from the requests.

Reply:
Added